test/commands/user/list.test.ts

```diff
@@ -62,93 +105,20 @@ describe('force:user:list', () => {
           },
         ],
       });
-  }
+  });
 
   test
-    .do(async () => {
-      await prepareStubs();
-    })
     .stdout()
     .command(['force:user:list', '--json', '--targetusername', 'testUser', '--targetdevhubusername', 'devhub@test.com'])
-    .it('should display the correct information from the default user', (ctx) => {
-      // testUser1@test.com is aliased to testUser
-      const expected = [
-        {
-          defaultMarker: '',
-          alias: 'testAlias',
-          username: 'testuser@test.com',
-          profileName: 'Analytics Cloud Integration User',
-          orgId: 'abc123',
-          accessToken: 'accessToken',
-          instanceUrl: 'instanceURL',
-          loginUrl: 'login.test.com',
-          userId: '0052D0000043PbGQAU',
-        },
-      ];
-      const result = JSON.parse(ctx.stdout).result;
-      expect(result).to.deep.equal(expected);
-    });
-
-  test
-    .do(async () => {
-      await prepareStubs();
-    })
-    .stdout()
-    .command([
-      'force:user:list',
-      '--json',
-      '--targetusername',
-      'testUser1@test.com',
-      '--targetdevhubusername',
-      'devhub@test.com',
-    ])
-    .it('should display the correct information from the default user with alias', (ctx) => {
-      // testUser1@test.com is aliased to testUser
-      const expected = [
-        {
-          defaultMarker: '',
-          alias: 'testAlias',
-          username: 'testuser@test.com',
-          profileName: 'Analytics Cloud Integration User',
-          orgId: 'abc123',
-          accessToken: 'accessToken',
-          instanceUrl: 'instanceURL',
-          loginUrl: 'login.test.com',
-          userId: '0052D0000043PbGQAU',
-        },
-      ];
+    .it('should display the correct information invoked with an alias', (ctx) => {
```

Comment:
Sure makes the test easier to understand with the expected data defined elsewhere, thank you, @mshanemc.